airbyte-integrations/connectors/destination-duckdb/integration_tests/spec.json

```diff
@@ -0,0 +1,31 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/duckdb",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Destination Duckdb",
+    "type": "object",
+    "required": ["destination_path"],
+    "additionalProperties": true,
+    "properties": {
+      "motherduck_api_key": {
+        "type": "string",
```

Comment:
Nit: giving those a title makes a prettier form (no underscores in the field names): 
<img width="309" alt="Screenshot 2023-09-15 at 17 54 06" src="https://github.com/airbytehq/airbyte/assets/1508364/17ffe388-e711-48bf-91dc-26b4c0047fed">

Reply:
Yeah - great catch. Thanks 👍